### PR TITLE
refactor: export command contents to script files

### LINKED ISSUE #21: Include RC009 in the review step inside the "config.yml"
### Orb version

v2.1.0

### What happened

In #20, the RC009 was included in the [exclusion list](https://github.com/CircleCI-Public/macos-orb/pull/20/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R19) of the orb tools review. This [review](https://github.com/CircleCI-Public/orb-tools-orb/blob/master/src/scripts/review.bats#L170) checks if there are commands with more than 64 characters (i.e. shell inside the yaml).

Orb development best practices dictate that shell should be inside `.sh` files, therefore, commands inside the yaml, should not contain code.

### Expected behavior

Commands with more than 64 characters of shell code should be extracted to their own `.sh` file and the RC009 review should not be excluded.

### PR DESCRIPTION
**Pull request by Jaryt**
closes #21

### CODE REVIEW

File: src/scripts/switch-ruby.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        echo "$ruby_version" > ~/.ruby-version
+    else
+        echo "chruby $ruby_version" >> ~/.bash_profile
+    fi
+else
+    version="$ORB_VAL_RUBY_VERSION"
+    needs_fuzzy_matching=$(echo "$ORB_VAL_RUBY_VERSION" | awk -F. '{print NF-1}')
+
+    if [[ $needs_fuzzy_matching == 1 && "$ORB_VAL_RUBY_VERSION" != "system" ]]; then
+        version=$(rbenv versions --bare | grep $test | head -n 1)
```
Comment (Jaryt) [inline src/scripts/switch-ruby.sh:26]: @BytesGuy I see "$test" but am not sure where it's being defined. Is this as expected?

File: src/commands/add-permission.yml [hunk 2]
```diff
@@ -9,19 +9,7 @@ parameters:
...
-              fi
-              sudo sqlite3 "/Library/Application Support/com.apple.TCC/TCC.db" "$tcc_update"
-              sudo sqlite3 "/Users/$USER/Library/Application Support/com.apple.TCC/TCC.db" "$tcc_update"
-          else
-              echo "Unable to add permissions! System Integrity Protection is enabled on this image"
-              echo "Please choose an image with SIP disabled. Documentation: https://circleci.com/docs/2.0/testing-macos"
-              exit 1
-          fi
+      environment:
+        ORB_VAL_BUNDLE_ID: << parameters.bundle-id >>
```
Comment (brivu) [inline src/commands/add-permission.yml:13]: Should these be `evaluated` in the `add-permission.sh` script? 

Is there a case where a user can use environment variables in these parameters?

**Comment by orb-publisher** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/macos?version=dev:d8663d630caa5e3eb3b15f844d1cbd3ee2e6bcda

File: src/scripts/switch-ruby.sh (hunk 1 above)
Comment (BytesGuy) [reply src/scripts/switch-ruby.sh]: @Jaryt Yeah that should be `$ORB_VAL_RUBY_VERSION`, must have been an artefact from local testing! Fixed in https://github.com/CircleCI-Public/macos-orb/pull/31/commits/1810e08bdd242a5e06b09eb1c17fc8b03e40740d

File: src/commands/add-permission.yml (hunk 2 above)
Comment (BytesGuy) [reply src/commands/add-permission.yml:13]: @brivu Can you clarify what you mean here? As far as I can tell this should all be fine

**Review by BytesGuy — APPROVED** [review]
Looks good to me 👍

**Comment by orb-publisher** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/macos?version=dev:255a83163787519b3747cf52a97d6641ffeca139

**Pull request merged**